fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -228,6 +232,17 @@ dropFrontendClause
    : DROP (FOLLOWER | OBSERVER) string
    ;
 
+addComputeNodeClause
+   : ADD (FREE)? COMPUTE NODE (TO identifier)? hostPortPairs          #addComputeNode
```

Comment:
because now the command of add be is alter system add backend [free] backend [to clusterName] hostPortPairs;
for unity here add free and cluster ; and the free and cluster is same as them in be